src/platform/forms-system/src/js/web-component-fields/VaRadioField.jsx

```diff
@@ -28,18 +30,37 @@ function optionsList(schema) {
  * ```js
  * exampleRadio: radioSchema(['cat', 'dog', 'octopus'])
  * ```
+ *
+ * Usage for an optional radio pattern:
+ *
+ * ```js
+ * exampleRadio: radioOptionalUI({
+ *  title: 'Select animal',
+ *  labels: {
+ *      dog: 'Dog',
+ *      cat: 'Cat',
+ *      octopus: 'Octopus',
+ *  }
+ * })
+ * ```
+ *
+ * Usage optional schema:
+ * ```js
+ * exampleRadio: radioOptionalSchema()
+ * ```
+ *
  * @param {WebComponentFieldProps} props */
 export default function VaRadioField(props) {
   const mappedProps = vaRadioFieldMapping(props);
-  const enumOptions =
-    Array.isArray(props.childrenProps.schema.enum) &&
-    optionsList(props.childrenProps.schema);
   const labels = props.uiOptions?.labels || {};
+  const { schema } = props.childrenProps;
+  // For non-required options, schema enum should be left undefined
+  const enumOptions =
```

Comment:
now that you've enhanced this to look at the uiSchema labels, could we just have the schema always be a string from our default radioSchema, and get rid of the enum part? 
seems like radioUI and radioSchema could then be used for both required and optional both in exactly the same way? Or is it more nuanced than that?

Reply:
yeah, I can do that!

Reply:
Then I kinda feel like we won't need the `radioOptionalUI` and `radioOptionalSchema`?

Reply:
Ok, I need to re-think this... everything works fine until you hit the review & submit page; then I see a `enumOptions.find is not a function` error (`RadioWidget` uses the same code as `SelectWidget`)

<img width="513" alt="console error showing 'Uncaught TypeError: enumOptions.find is not a function at SelectWidget (widgets.jsx line 25)'" src="https://github.com/user-attachments/assets/3c051716-f8d5-49e1-8b48-f0349944312f">